tasks.py

```diff
@@ -0,0 +1,17 @@
+from invoke import task, run
+
+@task
+def demofiles(ctx, clean=False):
+	if clean:
+		ctx.run('rm -rf demofiles')
+	ctx.run('mkdir demofiles')
```

Comment:
Let's put the repos in a python list at the top of the file and iterate over them in the function. Makes it easier to extend.